sdk/containerregistry/azure-containerregistry/samples/async_samples/sample_create_client_async.py

```diff
@@ -0,0 +1,51 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_create_client_async.py
+
+DESCRIPTION:
+    These samples demonstrate creating a ContainerRegistryClient and a ContainerRepositoryClient
+
+USAGE:
+    python sample_create_client_async.py
+
+    Set the environment variables with your own values before running the sample:
+    1) AZURE_CONTAINERREGISTRY_URL - The URL of you Container Registry account
+"""
+
+from dotenv import find_dotenv, load_dotenv
+import os
+
+class CreateClients(object):
+
+    def __init__(self):
+        load_dotenv(find_dotenv())
+        self.account_url = os.environ["AZURE_CONTAINERREGISTRY_URL"]
+
+    def create_registry_client(self):
+        # Instantiate the ContainerRegistryClient
+        # [START create_registry_client]
+        from azure.containerregistry.aio import ContainerRegistryClient
+        from azure.identity import DefaultAzureCredential
+        client = ContainerRegistryClient(self.account_url, DefaultAzureCredential())
```

Comment:
I believe you should be able to share the credential by declaring a `self.credential` in `__init__` as well (not a big deal though):
```suggestion
        from azure.identity.aio import DefaultAzureCredential
        client = ContainerRegistryClient(self.account_url, DefaultAzureCredential())
```